bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckablePullRequest.java

```diff
@@ -79,13 +79,16 @@ private String commitMessage(Hash head, List<Review> activeReviews, Namespace na
         var reviewers = reviewerNames(eligibleReviews, namespace);
         var currentUser = pr.repository().forge().currentUser();
 
-        if (manualReviewers) {
-            var allReviewers = reviewerNames(activeReviews, namespace);
-            var additionalReviewers = Reviewers.reviewers(currentUser, comments);
-            for (var additionalReviewer : additionalReviewers) {
-                if (!allReviewers.contains(additionalReviewer)) {
-                    reviewers.add(additionalReviewer);
+        if (manualReviewersAndStaleReviewers) {
+            reviewers.addAll(Reviewers.reviewers(currentUser, comments));
+            if (ignoreStaleReviews) {
+                var staleReviews = new ArrayList<Review>();
+                for (var review : activeReviews) {
+                    if (review.verdict() == Review.Verdict.APPROVED && !eligibleReviews.contains(review)) {
+                        staleReviews.add(review);
+                    }
                 }
+                reviewers.addAll(reviewerNames(staleReviews, namespace));
```

Comment:
I think all of this could be replaced with just adding `manualReviewersAndStaleReviewers` to the filter condition at line 76.

Reply:
Or we can add `manualReviewersAndStaleReviewers` to the filter condition at line 74,75,76